src/applications/edu-benefits/10203/components/ExitApplicationButton.jsx

```diff
@@ -12,11 +12,7 @@ export const ExitApplicationButton = ({ formId, isLoggedIn }) => {
   };
 
   return (
-    <a
-      className={'usa-button-primary va-button-primary'}
-      href="/education/"
-      onClick={onClick}
```

Comment:
When clicked, the behavior of this link changes the URL. This means that it should not be presented as a button so I have removed that style and kept it as an anchor tag.

Reference [Links vs Buttons](https://design.va.gov/components/button/#links-vs-buttons):

> A good rule is if the action changes the URL, it should not be a button.

> Button and link confusion can be very frustrating for assistive technology users. A user with a screen reader may pull up a list of links and may not find a specific link because it turns out that it’s been designated as a button in the markup.